app/controllers/cats_controller.rb

```diff
@@ -1,2 +1,30 @@
 class CatsController < ApplicationController
+  def index
+    cats = Cat.all
+    render json: cats
+  end
+
+  def create
+    # Create a new cat
+    cat = Cat.create(cat_params)
+    render json: cat
+  end
+
+  def update
+    cat = Cat.find(params[:id])
+    cat.update(cat_params)
+    render json:cat
+  end
+
+  def destroy
+    cat = Cat.find(params[:id])
+    cat.destroy
+  end
```

Comment:
This should work, but you may want to consider rendering the cat that you are destroying so there is some kind of a response from your controller.